packages/openapi-spec-builder/README.md

```diff
@@ -0,0 +1,73 @@
+# @loopback/openapi-spec-builder
+
+Builder API making it easy to create OpenAPI Spec/Swagger documents in your tests
```

Comment:
Making it easy to create OpenAPI (Swagger) specification documents in your tests using the builder pattern.